src/extensions/replay/sessionrecording.ts

```diff
@@ -97,8 +97,8 @@ type SessionRecordingStatus = 'disabled' | 'sampled' | 'active' | 'buffering' |
 export interface SnapshotBuffer {
     size: number
     data: any[]
-    sessionId: string
-    windowId: string
+    sessionId: string | null
```

Comment:
I'm not expecting session replay to work in cklsh mode at the moment, that would follow later

Reply:
If you aren't expecting it to work - why change the types here? Without a sessionId, replay should basically not run...

Reply:
yeah, i did quite a bit of rearranging so that I could make sure session id was never null. i don't want to lose that typecheck without adding safety back in. it might be fine here in this PR but could let a bug sneak in in a future PR

(disclaimer: i have not looked at the code just this thread)

Reply:
Cool, I've changed this to prevent the session id manager or session replay from loading when using this mode, and reverted the type changes